server/datastore/mysql/hosts.go

```diff
@@ -1906,3 +1906,47 @@ func (ds *Datastore) ListHostBatteries(ctx context.Context, hid uint) ([]*fleet.
 	}
 	return batteries, nil
 }
+
+func (ds *Datastore) MarkHostNotResponding(ctx context.Context, hostID uint) error {
+	// TODO: Would some logging be useful here?
```

Comment:
You already do some logging at the [service layer](https://github.com/fleetdm/fleet/pull/6436/files#diff-273e16dbbd725e4c93d924ba4062eb7de9318cf0fc9a9483b73448d1e95b99c3R513) - so I think we don't need to do any more logging at the data access layer